src/videojs-contrib-hls.js

```diff
@@ -289,6 +290,7 @@ class HlsHandler extends Component {
     if (tech.options_ && tech.options_.playerId) {
       let _player = videojs(tech.options_.playerId);
 
+      _player.gapSkipper();
```

Comment:
does videojs.gapSkipper also call this function? if so we should call gapSkipper somewhere else. Preferably towards the bottom of this constructor.